tests/flytekit/unit/core/test_type_hints.py

```diff
@@ -1553,18 +1547,6 @@ def t1() -> Foo:
     expected_struct.update({"x": 1, "y": "foo", "z": {"hello": "world"}})
     assert output_lm.literals["o0"].scalar.generic == expected_struct
 
-    @task
-    def t2() -> Bar:
-        return Bar(x=1, y={"hello": "world"}, z=Foo(x=1, y="foo", z={"hello": "world"}))
-
-    task_spec = get_serializable(OrderedDict(), serialization_settings, t2)
-    pt_map = TypeEngine.guess_python_types(task_spec.template.interface.outputs)
-    assert dataclasses.is_dataclass(pt_map["o0"])
-
-    output_lm = t2.dispatch_execute(ctx, _literal_models.LiteralMap(literals={}))
-    expected_struct.update({"x": 1, "y": {"hello": "world"}, "z": {"x": 1, "y": "foo", "z": {"hello": "world"}}})
-    assert output_lm.literals["o0"].scalar.generic == expected_struct
-
```

Comment:
We should remove this test case since it assumes `dict` uses `Dict[str, str]`.
In `marshmallow`, this is supported, but in `mashumaro`, this is not.

Reply:
is this breaking change?